docs/source/starter/new-project.rst

```diff
@@ -602,7 +602,7 @@ Here's an example adding a not-so-fancy learning rate decay rule:
                 group = [param_group['lr'] for param_group in optimizer.param_groups]
                 self.old_lrs.append(group)
 
-        def on_train_epoch_end(self, trainer, pl_module, outputs):
+        def on_train_epoch_end(self, trainer, pl_module):
```

Comment:
we did not have any warning in past that this removal will come?

Reply:
We did, see #7339